docker-compose.yml

```diff
@@ -240,11 +235,6 @@ services:
         depends_on:
             - engine-and-editor
             - parity-node0
-        healthcheck:
-            test: ["CMD", "curl", "-sS", "http://localhost"]
-            interval: 10s
-            timeout: 10s
-            retries: 60
```

Comment:
@kare is this removed because it should be redundant with the nginx check?

Reply:
@timoxley `HEALTHCHECK` defined in `docker-compose.yml` will override `HEALTCHECK` defined in `Dockerfile`. So `HEALTHCHECK` defined in `docker-compose.yml` is useful if `Dockerfile` doesn't define `HEALTHCHECK`